server/src/test/java/org/eclipse/openvsx/RegistryAPITest.java

```diff
@@ -1708,6 +1730,16 @@ private byte[] createExtensionPackage(String name, String version, String licens
     
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.

Reply:
Just tested this. Please add SimpleMeterRegistry.class to the @MockBean list and remove the UrlConfigService and MeterRegistry @bean methods.